src/ray/raylet/raylet_client.cc

```diff
@@ -428,4 +428,14 @@ Status raylet::RayletClient::ReturnWorker(int worker_port, const WorkerID &worke
       });
 }
 
+Status raylet::RayletClient::PinObjectIDs(const rpc::Address &caller_address,
+                                          const std::vector<ObjectID> &object_ids) {
+  rpc::PinObjectIDsRequest request;
```

Comment:
Can we feature flag distributed ref counting? I think it would just need to make this function a no-op right?